docs/background.md

```diff
@@ -2,22 +2,89 @@
+28.	  McGarry, The Rising, 163.
+29.	  McGarry, The Rising, 163.
+30.	  McGarry, The Rising, 162.
+31.	  McGarry, The Rising, 162, 165.
+32.	  McGarry, The Rising, 165.
+33.	  McGarry, The Rising, 259-260.
+34.	  McGarry, The Rising, 261.
+35.	  McGarry, The Rising, 161.
+36.	  McGarry, The Rising, 160.
+37.	  Keith Jeffrey, The GPO and the Easter Rising (Dublin, 2006), 47, 180.
+38.	  McGarry, The Rising, 160.
+39.	  McGarry, The Rising, 159.
+40.	  McGarry, The Rising, 160-161.
 
-As you discuss different authors, you may site them using (author, page) style parenthetical notation. Make sure that a full citation in [Chicago](http://chicagomanualofstyle.org) is added to the 'docs/credits.md' page.
 
 ---
 
 ## Digital Background
 
-You should also make sure to do research on any relevant digital work, whether scholarly articles or digital projects. Make sure to check Digital Humanities Quarterly, or [DHQ](http://www.digitalhumanities.org/dhq/), [Debates in the Digital Humanities](http://dhdebates.gc.cuny.edu/), [JStor](https://jstor.org), blogs and more to find relevant work.
+The Easter Rising of 1916 has recently celebrated the 100th anniversary of the failed rebellions attempt to overthrow British rule. The centennial event, celebrated in Ireland, led to many colleges and institutions trying to create digital projects to commemorate the rebellion. There is a myriad of ways in which these projects represent some type of digital scholarship on the rebellion. Some institutions made simple maps and pictures concerning the events surrounding the rebellion while others, collected a massive number of primary sources that have been digitized to be accessed by the public.
 
-You don't actually have to have a sepearte 'Digital Background' section. If it feels more natural to you, you may combine them into a single discussion. This is especially a good idea if you feel that with your topic you cannot talk about historiography without also talking about digital scholarship, and vice versa.
+Institutions like the Glasnevin Trust and ArcGIS made lists of those who died with additional information for the sake of either just having a comprehensive list or for the sake of creating a tutorial for how to make maps using their program (ArcGIS). Boston College, the National Library of Ireland, and the Irish times created databases for access to literature and speeches related to the Rebellion. The Irish Times shows the importance of the 1916 Easter Rising both contemporary to the Rebellion and now. The Boston College project for the Easter Rising in collaboration with segments of the Irish government and The Department of Culture, Heritage and the Gaeltacht, contains the most amount of information in one place along with interviews with historians and with readings of the literature of the Rebellion. The National Library of Ireland has collected the works of the biggest names of the Rebellion and have links to PDF scans of their writings.
```

Comment:
Why not add links to these projects and institutions, such as Glasnevin trust, et. al.? Again, think of it somewhat like citing.

In addition, is there no 'digital historiography' related to your approaches? If you are employing visualization, what about any of the debates about the validity of using visualizations for historical insights? This is where you could add intellectual heft to the methods you employ.